projects/storefrontlib/src/cms-components/user/login-register/login-register.component.ts

```diff
@@ -16,6 +16,7 @@ export class LoginRegisterComponent implements OnInit {
 
   ngOnInit(): void {
     if (this.checkoutConfigService.isGuestCheckout()) {
+      console.log('true');
```

Comment:
line 18 (above)

I don't think we need the flag for this anymore.

I don't remember the context for guest checkout exactly, but since this is cms-driven, I believe we don't need to have that config flag anymore too

Reply:
We still need the config flag for guest checkout. This part is not fully CMS driven.

Reply:
1. I added a PR to the sample data, changed the page status.

2. I looked into it a lot. This happens on all catalogs on my local. It's not the personalization, I changed it out and nothing happened. All dev servers use the PROFILE consent.

**One thing I noticed is that on dev4 we have the same sample data but not this problem.** It might mean our local has something wrong.